post-processor/vagrant/libvirt.go

```diff
@@ -68,7 +68,7 @@ func (p *LibVirtProvider) Process(ui packersdk.Ui, artifact packersdk.Artifact,
 
 	// Copy the disk image into the temporary directory (as box.img)
 	for _, path := range artifact.Files() {
-		if strings.HasSuffix(path, "/"+diskName) {
+		if strings.HasSuffix(path, diskName) {
```

Comment:
I wonder if we could use `filename.Base` here instead of `HasSuffix`? Since we're expecting the name of the disk to be the last part of the path, we can use this and do an equality check on that.
```suggestion
		if filepath.Base(path) == diskName {
```

Reply:
Good suggestion. I tried it out and made another commit.